vcs/src/main/java/org/openjdk/skara/vcs/git/GitRepository.java

```diff
@@ -38,17 +38,22 @@
 import java.util.stream.Collectors;
 
 public class GitRepository implements Repository {
-    public final static Map<String, String> NO_CONFIG_ENV = Map.of(
+    private final static Map<String, String> NO_CONFIG_ENV = Map.of(
             "HOME", "/this-does-not-exist-and-if-you-create-it-you-are-in-trouble",
             "XDG_CONFIG_HOME", "/this-does-not-exist-and-if-you-create-it-you-are-in-trouble",
             "GIT_CONFIG_NOSYSTEM", "true"
     );
 
+    public static Map<String, String> currentEnv = Collections.emptyMap();
     private final Path dir;
     private final Logger log = Logger.getLogger("org.openjdk.skara.vcs.git");
     private Path cachedRoot = null;
     private static final Hash EMPTY_TREE = new Hash("4b825dc642cb6eb9a060e54bf8d69288fbee4904");
 
+    public static void ignoreUserConfiguration(boolean ignore) {
```

Comment:
I would probably have named the method `ignoreConfiguration` and skipped the `boolean ignore` parameter. It doesn't seem like we ever pass `false` to `ignoreUserConfiguration`. If we need it in then in the future then we can add a method like `enableConfiguration`.